finder-app/Makefile

```diff
@@ -0,0 +1,57 @@
+
+# Author: Bhakti Ramani
+# Date: 2025-01-26
+# University: University of Colorado Boulder
+# Subject: ECEN 5713 Advanced Embedded Linux Development
+#
+# File Description:
+# Simple Makefile, generates executable for writer.c file using native
+# build and supports cross compilation of aarch64-none-linux-gnu
+# to run normally, type make
+# to use cross compilation, type make CROSS_COMPILATION=aarch64-none-linux-gnu-
+# Executable will generate using this perticular cross compilation but will
+# not be able to run on x86 pc. 
+# Logs status on fileresult.txt file
+#
+# File Name: Makefile
+
+ifeq ($(origin CROSS_COMPILE), undefined)
```

Comment:
@BhaktiRamani there's a simpler way to do this check, will discuss in class tomorrow.